vtexdocs/devportal
# fix: don't send feedback if it's a test

#### What is the purpose of this pull request?

To prevent feedbacks generated by the cypress test to be stored in the feedback spreadsheet.

#### What problem is this solving?

The amount of feedback generated by tests was making it difficult to visualize the real feedbacks.

#### How should this be manually tested?

Check the spreadsheet for feedbacks with the message "cypress-test", if there is none, the feedbacks generated by tests are not being stored as expected. And check if other feedbacks are still being stored in the spreadsheet.

#### Screenshots or example usage

#### Types of changes
